autopush/endpoint.py

```diff
@@ -174,7 +175,7 @@ def chid(self):
     @chid.setter
     def chid(self, value):
         """Set the ChannelID and record to _client_info"""
-        self._chid = value
+        self._chid = normalize_id(value)
         self._client_info["channelID"] = value
```

Comment:
We log the client_info["channelID"], but this sets that to value, the non-normalized one. It should prolly set it to self._chid which is now normalized.